lib/DDG/Spice/JrDevJobs.pm

```diff
@@ -1,18 +1,17 @@
 package DDG::Spice::JrDevJobs;
 # ABSTRACT: Search for Junior Developer Jobs from JrDevJobs.com
 
+use strict;
 use DDG::Spice;
 
-triggers start => qw(jrdevjobs);
+triggers startend => qw(jrdevjobs);
 
 spice to => 'https://www.jrdevjobs.com/jobs.json?query=$1';
 spice is_cached => 1;
 spice wrap_jsonp_callback => 1;
 
 handle remainder => sub {
-   s/(jobs)//;
-   s/(job)//;
-   s/(developer)//;
+    s/(jobs?|developer)//i;
 
```

Comment:
Let's add `use Text::Trim` to the top, and strip and trailing spaces before sending it to your API:

`return trim($_) if $_;`